.github/workflows/terragrunt-plan-production.yml

```
@@ -51,15 +51,28 @@ jobs:
         with:
           is-tagged: ${{ !startsWith(github.head_ref, 'release-please--') }} # If not the release PR branch, assume it is a revert PR
 
+  generate-lambda-functions-matrix:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - name: Generate matrix
+        id: generate-matrix
+        run: |
+          LAMBDA_ARRAY=$(echo $(cat ./.github/production-lambda-functions.conf) | sed 's/ //g' )
+          echo "::set-output name=matrix::$LAMBDA_ARRAY"
```

Just a minor comment there that GitHub is moving to this output format:
```suggestion
          echo "matrix=$LAMBDA_ARRAY" >> $GITHUB_OUTPUT
```

---

Gonna change it and quickly test before requesting approval.